index.md

```diff
@@ -13,44 +13,36 @@ will learn how to:
   a remote service
 - write an automated integration test for your application
 
-The tests will use:
-
-- the [Mocha test framework](https://mochajs.org/)
-- the [Chai assertion library](http://chaijs.com/), specifically the [Chai
-  Behavior-Driven Development style assertions](http://chaijs.com/api/bdd/).
-- the [Sinon spy, stub, and mock object library](http://sinonjs.org/)
-
-Also, the code will make use of the [Promise
-type](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise),
-standard in JavaScript/ECMAScript 6 and Node.js versions 0.12.7 and up.
-Promises are the standard way to write asynchronous operations, and this
-exercise will try to introduce you to their effective use in the process (and
-to avoid some of the pitfalls!).
-
 ## Assumptions
 
 This exercise assumes you are comfortable executing commands in a UNIX
 environment. Specifically, it expects that you are familiar with the basics of
 how to create directories and files on the command line of a terminal program.
 
-This exercise also assumes that you have already installed the latest version
-of Node.js to your machine, which is 5.1.0 at the time of writing. See the
-[JavaScript section of our Development Environment Standardization
-Guide](https://pages.18f.gov/dev-environment-standardization/javascript/) for
-instructions on how to set up a Node.js development environment.
+This exercise also assumes that you have already installed Node.js version 4.2
+or greater and Ruby version 2.2.3 or greater. (Ruby is used by the `./go`
```

Comment:
I think this needs to be called out a bit more explicitly and prominently only because the whole purpose of this repo is unit testing and application building in Node.js, yet you have to install Ruby; some folks are not going to understand why (e.g., you're actually installing content managed by Jekyll, a Ruby app, in order to view and work through things locally).